cockroachcloud/serverless-faqs.md

```diff
@@ -79,11 +79,13 @@ Yes, your free cluster has been automatically migrated to {{ site.data.products.
 
 CockroachDB [automatically collects statistics](../{{site.current_cloud_version}}/cost-based-optimizer.html#table-statistics) in a background process when certain conditions are met (for example, when more than 20% of rows in a table are modified). The statistics are used by the cost-based optimizer to tune statements for higher performance.
 
-When automatic statistics collection starts your cluster may consume RUs above the 100 RUs per second baseline when your workload is otherwise consuming RUs below the baseline. You can [turn off automatic statistics collection](../{{site.current_cloud_version}}/cost-based-optimizer.html#enable-and-disable-automatic-statistics-collection-for-clusters) to avoid these RU bursts, but the cost-based optimizer may choose inefficient statement plans as it doesn't have access to the latest statistics.
+When automatic statistics collection starts you may see a spike in RU usage. You can [turn off automatic statistics collection](../{{site.current_cloud_version}}/cost-based-optimizer.html#enable-and-disable-automatic-statistics-collection-for-clusters) to avoid these RU bursts, but the cost-based optimizer may choose inefficient statement plans if it doesn't have access to the latest statistics.
+
+Refreshing the Cloud Console will also consume RUs.
 
 ### What is the cold start latency of a {{ site.data.products.serverless }} cluster?
 
-When a {{ site.data.products.serverless }} cluster is idle, it will scale down to zero and consume no RUs. When the cluster becomes active again it will begin serving requests within a fraction of a second, typically around 600 milliseconds.
+When a {{ site.data.products.serverless }} cluster is idle, it will scale down to zero and consume no RUs. When the cluster becomes active again it will begin serving requests within a fraction of a second, typically around 600 milliseconds. Note that cold starts consume RUs, so you may see a spike in usage.
```

Comment:
I find myself wondering what it scales down to zero of. I guess it's RUs, since it doesn't really have nodes? What about "will scale down and consume no RUs"? I'm less familiar with Serverless so ignore this if I'm misunderstanding.

Reply:
> wondering what it scales down to zero of. 

Good point. I dont think users are aware that scale ups and downs are in reference to sql server instances (kubernetes pods). You could also think of it as scale to 0 meaning 0 CPUs.

Reply:
Thanks for clarifying, it makes sense. I guess that it's difficult to observe an idle cluster that is consuming no RUs, if viewing its pages in Cloud Console consumes RUs.